### PR TITLE
Remove coolant control

### PR DESCRIPTION
**Pull request by StephanCoetzee**
Remove coolant control from the code to clean it up.

### CODE REVIEW

File: gcode.c [hunk 1]
```diff
@@ -301,16 +300,6 @@ uint8_t gc_execute_line(char *line)
          #ifdef ENABLE_M7  
           case 7:
          #endif
```
Comment (penguinland) [inline gcode.c]: Remove this part about `ENABLE_M7`, too. That was a coolant option.

File: gcode.c [hunk 2]
```diff
@@ -1022,7 +1005,6 @@ uint8_t gc_execute_line(char *line)
    group 6 = {M6} (Tool change)
    group 7 = {G40, G41, G42} cutter radius compensation
    group 8 = {G43} tool length offset (But G43.1/G94 IS SUPPORTED)
-   group 8 = {*M7} enable mist coolant
```
Comment (penguinland) [inline gcode.c:1025]: There used to be 2 different group 8's. Is this documentation correct?

**Review by penguinland — COMMENTED** [review]
I think this PR is missing corresponding changes to cpu_map_keyme2560.h. but you know this better than I do; I might be wrong.

File: gcode.c (hunk 2 above)
Comment (Jeff-Ciesielski) [reply gcode.c:1025]: Yes, it's correct.  If you look here: http://dynomotion.com/Help/GCodeScreen/EMC_Handbook/node65.html, the modal group numbers are duplicated (one set for 'G' commands, one set for 'M' commands).

**Comment by Jeff-Ciesielski** [comment]
@StephanCoetzee, Can you confirm that this still builds?  If so, LGTM

**Comment by StephanCoetzee** [comment]
It builds

**Comment by penguinland** [comment]
The cool ant is now out of control.

cool_ant_by_cotton_gravy-d4mrqax

**Pull request merged**